drivers/include/saul/periph.h

```diff
@@ -32,13 +32,20 @@ extern "C" {
 #endif
 
 #ifdef MODULE_SAUL_GPIO
+typedef enum {
+    SAUL_GPIO_INVERTED = 1 << 0,  /**< pin is used as inverted */
+    SAUL_GPIO_CLEAR    = 1 << 1,  /**< set pin inactive after initialization */
```

Comment:
Maybe rename to `SAUL_GPIO_INIT_CLEAR` to be more explicit. Same with `SAUL_GPIO_INIT_SET` below